### PR TITLE
Updated Note to add RTF and PCE

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Java API::
+Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
+
+JNI (Java Native Interface)::
+JNI is a layer in the SAP Connector library. JCo wraps the C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
+
+RFC (remote function call)::
+Communication with the SAP system is performed over the RFC protocol. RFC means calling BAPI or triggering IDoc processing that runs in another system as a calling program. The RFC interface enables function calls between two SAP systems or between the SAP and external system.
+
+RFC Layer::
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]:
```suggestion
RFC layer::
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+SAP component that processes remote function calls.
+
+RFC Library::
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]:
```suggestion
RFC library::
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+SAP Java IDoc class library::
+Provides structured, high-level interpretation and navigation of SAP IDocs in Java that consists of the following add-on packages:
+** SAP Java base IDoc class library: a middleware-independent library that provides a set of general base classes and interfaces for middleware dependent Java IDoc class library implementations.
+** SAP Java connector IDoc class library: A middleware-independent library for creating, sending, and receiving IDocs.
+
+SAP NetWeaver::
+One of the main technologies and application platforms used by SAP solutions. The main component of NetWeaver is the SAP Web Application Server (WebAS), which provides the runtime environment for SAP applications like ERP, CRM, SCM, PLM, SRM, and BI. Other components include the enterprise portal, exchange infrastructure, master data management, and mobile infrastructure. The SAP NetWeaver is an umbrella term for these technical components. SAP NetWeaver runs on both the Java and ABAP stacks.
+
+SNC (Secure Network Communications)::
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc:51]: I see both "Secure" and "Secured" when I search online. What do you think?

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+ABAP (Advanced Business Application Programming)::
+SAP proprietary programming language and part of the NetWeaver platform for building business applications.
+
+ALE (Application Link Enabling)::
+Sets up distributed applications and integrated installation of SAP systems. This involves business-driven message exchange using consistent data across loosely linked SAP apps. Apps are integrated through synchronous and asynchronous communication, rather than by use of a central database.
+
+BAPI (Business Application Programming Interface)::
+The function module that fulfills design criteria, such as:
+** Implement a method of a SAP business object
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc:16]: I would change each of these to gerund forms: Implementing, Maintaining, and so on.

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+** Maintain a static interface through different versions of the SAP system
+** Enable remotely
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc:18]: This begs for an object: enabling what remotely? Or do we want "Remote enablement"?

Comment (kbostrom) [reply modules/ROOT/pages/sap/sap-connector-terminology.adoc:18]: ...or even Providing you the ability to...do these things?

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+** Handle errors
+
+FM (function module)::
+Procedures defined in the ABAP language of SAP that allow the encapsulation and reuse of global functions in SAP.
+
+IDoc (intermediate document)::
+Standard SAP format for electronic data interchange between SAP systems. Different message types (such as delivery notes or purchase orders) generally correspond to different special formats, known as IDoc types. Multiple message types with related content can, however, be assigned to a single IDoc type.
+
+Java API::
+Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]: GLOBALLY replace "and/or" with the better of the two, per style ref.

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+JNI (Java Native Interface)::
+JNI is a layer in the SAP Connector library. JCo wraps the C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]: Different from other definitions by repeating the term: nec?
Also, should we define JCo?
```suggestion
JNI is a layer in the SAP Connector library. JCo wraps the C libraries in Java to provide platform-native access to the SAP system. RFC middleware uses the RFC library through the JNI layer.
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+RFC (remote function call)::
+Communication with the SAP system is performed over the RFC protocol. RFC means calling BAPI or triggering IDoc processing that runs in another system as a calling program. The RFC interface enables function calls between two SAP systems or between the SAP and external system.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]:
```suggestion
Communication with the SAP system is performed over the RFC protocol. A remote function call calls BAPI or triggers IDoc processing that runs in another system as a calling program. The RFC interface enables function calls either between two SAP systems or between the SAP and external system.
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+RFC Layer::
+SAP component that processes remote function calls.
+
+RFC Library::
+Libraries of C language-based functions to access the SAP system. The RFC library is addressed by JNI.
+
+SAP Java IDoc class library::
+Provides structured, high-level interpretation and navigation of SAP IDocs in Java that consists of the following add-on packages:
+** SAP Java base IDoc class library: a middleware-independent library that provides a set of general base classes and interfaces for middleware dependent Java IDoc class library implementations.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]: Java-based?
```suggestion
** SAP Java base IDoc class library: A middleware-independent library that provides a set of general base classes and interfaces for middleware-dependent Java IDoc class library implementations.
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+** SAP Java connector IDoc class library: A middleware-independent library for creating, sending, and receiving IDocs.
+
+SAP NetWeaver::
+One of the main technologies and application platforms used by SAP solutions. The main component of NetWeaver is the SAP Web Application Server (WebAS), which provides the runtime environment for SAP applications like ERP, CRM, SCM, PLM, SRM, and BI. Other components include the enterprise portal, exchange infrastructure, master data management, and mobile infrastructure. The SAP NetWeaver is an umbrella term for these technical components. SAP NetWeaver runs on both the Java and ABAP stacks.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]:
```suggestion
One of the main technologies and application platforms used by SAP solutions. The main component of NetWeaver is SAP Web Application Server (WebAS), which provides the runtime environment for SAP applications like ERP, CRM, SCM, PLM, SRM, and BI. Other components include the enterprise portal, exchange infrastructure, master data management, and mobile infrastructure. SAP NetWeaver is an umbrella term for these technical components. SAP NetWeaver runs on both the Java and ABAP stacks.
```

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+SNC (Secure Network Communications)::
+SNC integrates with the SAP NetWeaver single sign-on, or an external security product that works with SAP systems. SNC provides security functions that are not directly available with SAP systems.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-connector-terminology.adoc]: ditto re. renaming term:
```suggestion
Integrates with SAP NetWeaver single sign-on or an external security product that works with SAP systems. SNC provides security functions that are not directly available with SAP systems.
```

**Review by kbostrom — APPROVED** [review]
Done.

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc (hunk 3 above)
Comment (valkyrie69) [reply modules/ROOT/pages/sap/sap-connector-terminology.adoc:51]: Secure.

File: modules/ROOT/pages/sap/sap-connector-terminology.adoc (hunk 5 above)
Comment (valkyrie69) [reply modules/ROOT/pages/sap/sap-connector-terminology.adoc:18]: Yes, it's remote enablement. I'll figure out a way to reword.

**Pull request merged**